content/en/contribute/code/android/releasing.md

```diff
@@ -21,17 +21,17 @@ All Medic's android projects automatically build, sign, and release builds via G
 
 ## Alpha for release testing
 
-1. Make sure all issues for this release have passed AT and been merged into `master`. You can also create an alpha release from a feature branch, to provide the needed `.apk` files to the QA team.
-2. Create a git tag starting with `v` and ending with the alpha version, e.g. `git tag v1.2.3-alpha.1` and push the tag to GitHub (`git push --tags`). For features branches, you can add the name of the branch or whatever keyword helps to identify the the release, e.g. `v1.2.3-alpha.dark-theme.4`.
-3. Creating the tag will trigger the building and signing of the app in CI. The release-ready APKs are available for side-loading from the Releases section in the project (e.g. [CHT-Android Releases](https://github.com/medic/cht-android/releases)), along with the AABs that may be required by the Google Play Store. Note that the created release on GitHub with the generated artifacts will not be explicitly linked to the tag because the release is in draft state.
+1. Ensure all issues for this release have passed AT and been merged into `master`. You can also create an alpha release from a feature branch, to provide the needed `.apk` files to the QA team.
+2. Create a git tag starting with `v` and ending with the alpha version, e.g. `git tag v1.2.3-alpha.1`, and push the tag to GitHub (`git push --tags`). For features branches, you can add the name of the branch or whatever keyword helps to identify the release, e.g. `v1.2.3-alpha.dark-theme.4`.
+3. Creating the tag will trigger the building and signing of the app in CI. The release-ready APKs are available for side-loading from the Releases section in the project (e.g. [CHT-Android Releases](https://github.com/medic/cht-android/releases)), along with the AABs that the Google Play Store may require. Note that the created release on GitHub with the generated artifacts will not be explicitly linked to the tag because the release is in a draft state.
 4. **Side-Load**: for testing internally, or apps that are not published in the Play Store like Collect and Gateway: Navigate to the GitHub Releases page (linked above) and download the relevant APKs for distribution.
 5. Announce the release in _#quality-assurance_.
 
 
 ## Production release
 
-1. Repeat step 2-3 from the above [alpha release section](#alpha-for-release-testing) with the naming convention `v<major>.<minor>.<patch>`.
 1. Update the CHANGELOG with the details of what's in this release.
```

Comment:
Can you please fix the numeration, all these are `1.`?

Reply:
When rendering, markdown changes them to appropriate numbers according to the sequence.

Reply:
Oh interesting. I guess it's just OCD haha